src/plugins/intel_gpu/src/graph/graph_optimizer/prepare_padding.cpp

```diff
@@ -28,6 +30,20 @@ void prepare_padding::run(program& p) {
             if (node->get_preferred_impl_type() == impl_types::onednn)
                 continue;
 
+            auto& conv_input_node = node->get_dependency(0);
+            // Check previous node is one of padding support primitive or not
```

Comment:
This loop is needed to be done regardless the user. It is recalculating required padding from the output size.

Reply:
Could you check the benchmark check extensively? (for all of i8 and fp16/32)

Reply:
@yeonbok  I'm checking the performance regression check about all int8 and fp32/16 models. About first comment, the loop is checking users of previous node. Then how can I make the loop without considering user?